feature-libs/commerce-quotes/components/commerce-quotes-action-links/commerce-quotes-action-links.module.ts

```diff
@@ -0,0 +1,21 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { I18nModule, UrlModule } from '@spartacus/core';
+import { CommerceQuotesActionLinksComponent } from './commerce-quotes-action-links.component';
+
+@NgModule({
+  imports: [CommonModule, I18nModule, RouterModule, UrlModule],
+  declarations: [CommerceQuotesActionLinksComponent],
+  exports: [CommerceQuotesActionLinksComponent],
+})
+
+/**
+ * TODO: Think about the way how component can be initialised,
+ * should does it be separate CMS Component and render it on the page
+ * by using CMS mapping?
+ *
+ * Or render action links directly in any components template by
+ * using component's selector name.
+ */
```

Comment:
I believe it could be separate CMS Component. We have on mockups some kind of header with quote name and I could be part of this CMS header component (maybe nesting of so called `QuoteDetailsHeaderComponent`). @psmul please share your thoughts.

Reply:
No, I don't think we need CMS mapping for this component.